caom2utils/caom2utils/fits2caom2.py

```diff
@@ -3894,18 +4033,23 @@ def _visit(plugn, parser, obs, visit_local, **kwargs):
                 kwargs['headers'] = parser.headers
                 if visit_local is not None:
                     kwargs['fqn'] = visit_local
+                if product_id is not None:
+                    kwargs['product_id'] = product_id
                 try:
-                    if plgin.update(observation=obs, **kwargs):
+                    result = plgin.update(observation=obs, **kwargs)
```

Comment:
plugn is the string name, plgin is the python pointer for execution. Should they be renamed?

Reply:
Sorry. My bad - I didn't notice there's a collapsed code section in the diff and I've assumed it was just a spelling error. I would have used `plugin_name` in the method argument because it's more obvious when the function does not have documentation.

Reply:
Names changed in the code now.